apps/client/src/components/Question.js

```diff
@@ -116,7 +117,9 @@ const Question = ({
         answers={answers}
         userAnswer={userAnswer}
       />
-      {questionNeedsPermit && <PermitAlert />}
+      {questionNeedsPermit && (
```

Comment:
Shouldn't this be `(questionNeedsPermit || questionNeedsContactExit) &&`?

Reply:
I thought the same thing when I started building it, but it turned out that if there's a Contact Conclusion that `questionNeedsPermit` is `true`. So that's why we don't need that.

Reply:
Then I think the name `questionNeedsPermit` is very misleading.

Reply:
Done